src/converters/notion/index.ts

```diff
@@ -0,0 +1,232 @@
+import { parse } from 'csv-parse/sync';
+import { TanaIntermediateAttribute, TanaIntermediateFile, TanaIntermediateNode } from '../../types/types';
+import { idgenerator } from '../../utils/utils';
+import { IConverter } from '../IConverter';
+
+const isPageLink = (s: string) => {
+    return !!s.match(/\.md$/)
+}
+
+const parseFileContents = (fileContent: string): Record<string, string>[] => {
+    // Looks like notion exporter inserts a Zero Width No-Break Space character at the start of the file.
+  return parse(fileContent.slice(1), {columns: true, skip_empty_lines: true, encoding: 'utf8', })
+}
```

Comment:
using `csv-parse` to extract the contents. Im not married to this package if theres a better one out there.